aws/resource_aws_workspaces_directory.go

```diff
@@ -120,8 +153,8 @@ func resourceAwsWorkspacesDirectoryCreate(d *schema.ResourceData, meta interface
 
 	input := &workspaces.RegisterWorkspaceDirectoryInput{
 		DirectoryId:       aws.String(directoryId),
-		EnableSelfService: aws.Bool(false), // this is handled separately below
-		EnableWorkDocs:    aws.Bool(false),
+		EnableSelfService: aws.Bool(true), // this is handled separately below
```

Comment:
not familiar with workspapces, whats the reason these are change to true?

Reply:
[`EnableSelfService`](https://docs.aws.amazon.com/workspaces/latest/api/API_RegisterWorkspaceDirectory.html#WorkSpaces-RegisterWorkspaceDirectory-request-EnableSelfService) attribute determines self-service capabilities enable/disable status for all workspaces in the particular workspaces directory. For example, with enabled self-service permissions, workspaces administrator can allow workspace user (client) to change the compute type or volume size without administrator's involvement (on-demand).

Here I've changed it from `false` to `true` for the same reason as `EnableWorkDocs` attribute below – I thought it couldn't be changed after directory creation. However, self-service permissions can be changed both with [`ModifySelfservicePermissions `](https://docs.aws.amazon.com/workspaces/latest/api/API_ModifySelfservicePermissions.html) API call and AWS Management Console at any time. I'll return it back to `false` to preserve backward compatibility. Thank you for your thorough review.

P.S. If you `EnableSelfService` attribute it to `false`, not all attributes behind it are set to `false` to.